k-distribution/tests/regression-new/haskell-backend-kprove/verif.k

```diff
@@ -7,12 +7,19 @@ module VERIF-SYNTAX
     rule foo => 3 [macro]
 
     syntax Pgm ::= doIt ( Int )
+    syntax Pgm ::= addCounter ( Int )
 endmodule
 
 module VERIF
     imports VERIF-SYNTAX
 
-    configuration <k> $PGM:Pgm </k>
+    configuration <k> $PGM:Pgm </k> <counter> 1 </counter> <sum> 0 </sum>
+
+    rule addCounter(0) => .
```

Comment:
```suggestion
    rule <k> addCounter(0) => . ... </k>
```

I want to get people out of the habit of omitting `<k>` tags. It makes it quite confusing in definitions which use a lot of `function`s.